renderer.js

```diff
@@ -99,6 +142,11 @@ class Song {
 }
 
 function playSong(e) {
+
+    if (e.target.classList.contains('btnDownload')) {
```

Comment:
```js 
const btnDownload = document.querySelector('.btnDownload');
```
should be done before, I think .
and I think you need to compare to `e.currentTarget` but not sure